drivers/sensor/bmp388/Kconfig

```diff
@@ -12,6 +12,12 @@ menuconfig BMP388
 
 if BMP388
 
+config BMP388_VALIDATE_TIMINGS
```

Comment:
Based on your problem description it sounds like we don't want this to be optional. Any reason someone wouldn't want these checks?

Reply:
Well, usually after you're done developing, you don't touch OSR/ODR any more. It's mostly useful as a "fault finding" measure while writing code, but mostly useless in production. And of course the checks do add code space and some processing to each attribute setting or power mode switch.

Reply:
Maybe it would be useful to rename it `BMP388_DEBUG_TIMINGS` and to enable it by default? That way it's on when you start developing, but it's clear that it's probably safe to remove from production builds?

Reply:
Since the driver supports setting ODR/OSR dynamically I feel like it should definitely be enabled by default. Current name seems fine.

Reply:
Cool, I will make it enabled by default